observations/obs_converters/AIRS/convert_amsu_L1.rst

```diff
@@ -1,83 +1,103 @@
+AMSU-A level 1A digital numbers (DN) and contains 15 microwave channels in the
+50-90 GHz and 23-32 GHz regions of the spectrum. A day's worth of data is divided
+into 240 scenes (granules), each of 6 minute duration. An AMSU-A scene contains 
+30 cross-track footprints in each of 45 along-track scanlines, for a total of 
+45 x 30 = 1350 footprints per scene. AMSU-A scans three times as slowly as AIRS 
+(once per 8 seconds) and its footprints are approximately three times as large as
+those of AIRS (45 km at nadir). This results in three AIRS scans per AMSU-A scans
+and nine AIRS footprints per AMSU-A footprint.
+
+For more details on the history of the AMSU/A satellite instrumentation
+see the following `link <https://en.wikipedia.org/wiki/Advanced_microwave_sounding_unit#History>`_.
+
+To summarize, AMSU/A was flown on satellites NOAA 15-17. Versions of AMSU-A also
+fly on the Aqua satellite (that also houses AIRS) as well as the European MetOp
+satellite. It has been replaced by the Advance Technology Microwave Sounder (ATMS)
+on the satellite NOAA-20.
+
+Instructions to download the AMSU-A L1B Version 5 (AIRABRAD) dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The AMSU-A data is located within the Goddard Earth Sciences Data and Information
+Services Center (GES DISC) `located here <https://disc.gsfc.nasa.gov/>`_. You need
+to create an Earthdata account before you can download data. To access the 
+AMSU-A data, search on keyword ``AIRABRAD`` and locate
+the **AIRS/Aqua L1B AMSU (A1/A2) geolocated and calibrated brightness temperatures V005
+(AIRABRAD)** heading within your search results. 
+
+Next, under the Data Access header, click on `Subset/Get Data`, then refine your
+search results by 1) data range (time) and 2) spatial region.
+
+There are various options for downloading, however,the most straightforward approach
+for macOS and Linux users is to use the ``wget`` command.  The ``download instructions``
+provide the proper wget flags/options.  The ``Download Links List`` provides
+the AMSU-A file list based on your search results.
+
+
+| Each granule is about 560K and has names like
+
+::
+
+   AIRS.2019.06.22.236.L1B.AMSU_Rad.v5.0.0.0.G19174110442.hdf
+
+Advanced Microwave Sounding Unit (AMSU-A) L1B Brightness Temperatures
+---------------------------------------------------------------------
+
+Perform the following steps to convert the AMSU_L1 observations:
+
+- 1. Download the `h4tonccf_nc4 tool <http://hdfeos.org/software/h4cflib.php>`_ provided 
+     from the hdf-eos website. Options are provided for Mac, Linux and Windows platforms. 
+
+- 2. Compile the the h4tonccf_nc4 tool following the instructions provided on the hdf-eos
+     site. Compiling requires both HDF4 and HDF-EOS2 libraries. For Derecho the HDF4
+     libraries are accessed through the ``module load hdf`` command.  HDF-EOS2 libaries are
+     provided at ``/glade/campaign/cisl/dares/libraries/``.     
```

Comment:
I need more guidance on building the h4tonccf_nc4 tool to perform the hdf-eos to netcdf conversion step.  On the website, the binary is provided for the Linux CentOS7 and macOS operating systems, but not sure if this is appropriate for Derecho with SUSE.  Also requires additional configuration file to build on autoconf system, so want to make sure I am headed down correct path.

Reply:
I downloaded the CentOS7, 
 wget https://hdfeos.org/software/h4cflib/bin/linux/v1.3/CentOS7/h4tonccf_nc4

Ran it on Derecho, seems ok:
```
chmod +x h4tonccf_nc4
./h4tonccf_nc4 
```
```
Usage: h4tonccf [hdf file name] or h4tonccf [hdf file name] [netcdf file name] 
Note: If the netcdf file name is not specified, the hdf file name will be used instead with .nc as extension.
```

```
./h4tonccf_nc4 AIRS.2022.01.01.120.L2.RetStd_IR.v6.0.33.0.G22235231529.hdf 
```
```
Done with writing netcdf file AIRS.2022.01.01.120.L2.RetStd_IR.v6.0.33.0.G22235231529.nc.
```